src/report/create_all_reports.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+"""Generate all reports."""
+
+# Standard Python Libraries
+import csv
+
+# generate_https_scan_report.py isn't written in a way that easily
+# allows it to be run in any other way, but Bandit doesn't like us to
+# use subprocess.  Hence the nosec.
+import subprocess  # nosec B404
+
+HOME_DIR = "/home/cisa"
+SHARED_DATA_DIR = HOME_DIR + "/shared/"
```

Comment:
I should have suggested this the first review just so we're using f-strings consistently.

```suggestion
SHARED_DATA_DIR = f"{HOME_DIR}/shared"
```